src/simulation/monsters/special/Hespori.ts

```diff
@@ -67,9 +67,14 @@ const MainTable = new LootTable()
 export class Hespori extends SimpleMonster {
 	public kill(quantity = 1, options: MonsterKillOptions = { farmingLevel: 99 }): ItemBank {
 		const loot = new Loot();
+		let farmingLvl;
```

Comment:
```suggestion
		let farmingLvl = options.farmingLevel ? options.farmingLevel : 99;
```

you can use a ternary to do what that `if` block you have is doing